pkgs/tools/misc/ckb/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub, libudev, pkgconfig, qtbase, qmakeHook, zlib }:
+
+stdenv.mkDerivation rec {
+  version = "0.2.6";
+  name = "ckb-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ccMSC";
+    repo = "ckb";
+    rev = "v${version}";
+    sha256 = "04h50qdzsbi77mj62jghr52i35vxvmhnvsb7pdfdq95ryry8bnwm";
+  };
+
+  buildInputs = [
```

Comment:
Thanks for the heads up. For future reference, what's the difference between `buildInputs` and `nativeBuildInputs`?